dist/cta-button/cta-button.css

```diff
@@ -90,13 +90,21 @@ span.cta-btn__cell--fixed-height svg.icon {
   overflow: visible;
   width: 1rem;
 }
+a.cta-btn--truncated,
+a.cta-btn--truncated span {
+  line-height: 1.4em;
```

Comment:
This seems like a bit of a random value. Does it need to be exactly this?

Reply:
Its being generated from `.btn-truncate();` thats being used for truncation.